repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -55,6 +57,14 @@ class ExecutorClassLoader(conf: SparkConf, classUri: String, parent: ClassLoader
     }
   }
 
+  override def getResource(name: String): URL = {
+    parentLoader.getResource(name)
```

Comment:
@holdenk maybe you have some thoughts on this?
Also summoning @vanzin

Doesn't this need to check `userClassPathFirst`?